Shared/Factories/GetMovieBackdropInteractorFactory.swift

```diff
@@ -0,0 +1,9 @@
+public class GetMovieBackdropInteractorFactory {
```

Comment:
By Clean Code a class name should represent what his real intent. With just `MovieBackdropInteractor` it is not possible to realize what it means. Maybe instead `Get` prefix would be better `Fetch` or `Find`.